Document/0x06b-Basic-Security-Testing.md

```diff
@@ -359,7 +351,29 @@ Possible values for the property [UIDeviceFamily](https://developer.apple.com/li
 
```

Comment:
As part of the installing (I know that this is not what the chapther should be about, but maybe add ios-deploy?)

Reply:
Will be covered here: https://github.com/OWASP/owasp-mstg/issues/1248